src/GitVersionCore/SemanticVersionPreReleaseTag.cs

```diff
@@ -95,7 +95,8 @@ public static SemanticVersionPreReleaseTag Parse(string preReleaseTag)
             var match = Regex.Match(preReleaseTag, @"(?<name>.*?)\.?(?<number>\d+)?$");
             if (!match.Success)
             {
-                Logger.WriteWarning($"Unable to successfully parse semver tag {preReleaseTag}");
+                // TODO check how to log this
+                Console.WriteLine($"Unable to successfully parse semver tag {preReleaseTag}");
```

Comment:
Why the change from `Logger` to `Console`?

Reply:
I left it for now like this as it was not easy to inject ILog into the static method. I will consider changing that later. There is a second TODO in Config.cs class